components/atom/backToTop/src/index.js

```diff
@@ -16,95 +16,83 @@ const STYLES = {
   LIGHT: 'light'
 }
 
-class AtomBackToTop extends PureComponent {
-  state = {
-    show: null,
-    hover: false
-  }
-
-  intervalId = 0
-  container = null
-
-  scrollStep = () => {
-    const {scrollTop} = this.container
-    const {scrollSteps} = this.props
-    const {intervalId} = this
+const AtomBackToTop = ({
+  iconTop: IconTop,
+  textTop,
+  scrollSteps,
+  scrollIntervalTime,
+  refContainer,
+  style
+}) => {
+  const [show, setShow] = useState(null)
```

Comment:
I see you're using show as boolean. We might want to put as initial value `false` as well.

Reply:
👍